cves/kernel/CVE-2018-20169.yml

```diff
@@ -360,9 +330,47 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+autodiscoverable:
+  note: |
+    Since the bug occurs from USB descriptor reading, testing requires 
+    a USB device that allows metadata to be modified. Once the testing 
+    device has been created, the device can be manually used to test 
+    other devices.
+  answer: false
+  instructions: |
+    Is it plausible that a fully automated tool could have discovered
+    this? These are tools that require little knowledge of the domain,
+     e.g. automatic static analysis, compiler warnings, fuzzers.
+
+    Examples for true answers: SQL injection, XSS, buffer overflow
+
+    In systemd, the actually use OZZ Fuzz. If there's a link to it, add it here.
+
+    Examples for false: RFC violations, permissions issues, anything
+    that requires the tool to be "aware" of the project's
+    domain-specific requirements.
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+interesting_commits:
+  commits:
```

Comment:
missing interesting commits entry

Reply:
No interesting commit found between vcc and fix